### PR TITLE
1990 RJSF e2e tests

### PR DESCRIPTION
**Pull request by jbalboni**
Adds schema and e2e tests for new 1990 form.

### CODE REVIEW

File: test/e2e/edu-1990-rjsf-helpers.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    .fillDate('root_view:benefitsRelinquishedContainer_benefitsRelinquishedDate', data['view:benefitsRelinquishedContainer'].benefitsRelinquishedDate);
+}
+
+function completeBenefitsSelection(client, data) {
+  const benefits = data['view:selectedBenefits'];
+  client
+    .clickIf('input[name="root_view:selectedBenefits_chapter33"]', benefits.chapter33)
+    .clickIf('input[name="root_view:selectedBenefits_chapter30"]', benefits.chapter30)
+    .clickIf('input[name="root_view:selectedBenefits_chapter1606"]', benefits.chapter1606)
+    .clickIf('input[name="root_view:selectedBenefits_chapter32"]', benefits.chapter32);
```
Comment (cvalarida) [inline test/e2e/edu-1990-rjsf-helpers.js]: Was `clickIf` causing IE sauce labs failures? Do Remember that right?

**Review by cvalarida — APPROVED** [review]

File: test/e2e/edu-1990-rjsf-helpers.js (hunk 1 above)
Comment (jbalboni) [reply test/e2e/edu-1990-rjsf-helpers.js]: That sounds vaguely true? I will run these in sauce and see.

File: test/edu-benefits/1990-rjsf/schema/maximal-test.json [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    "view:otherContactInfo": {
+      "email": "test@test.com",
+      "view:confirmEmail": "test@test.com",
+      "homePhone": "234243244444",
+      "mobilePhone": "234234244444"
+    },
+    "educationProgram": {
+      "name": "UMASS Amherst",
+      "address": {
+        "street": "230 1st st",
```
Comment (cehsu) [inline test/edu-benefits/1990-rjsf/schema/maximal-test.json]: Is there a street field for this form?

File: test/edu-benefits/1990-rjsf/01-all-fields.e2e.spec.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+    // rotc history
+    client
+      .waitForElementVisible('label[for="root_seniorRotcScholarshipProgram"]', Timeouts.slow);
+    Edu1990Helpers.completeRotcHistory(client, testData.data);
+    client
+      .axeCheck('.main')
+      .click('.form-progress-buttons .usa-button-primary');
+    E2eHelpers.expectNavigateAwayFrom(client, '/military-history/rotc-history');
+    // contributions history
```
Comment (cehsu) [inline test/edu-benefits/1990-rjsf/01-all-fields.e2e.spec.js]: An extra blank line between here could be nice.

File: test/e2e/edu-1990-rjsf-helpers.js (hunk 1 above)
Comment (jbalboni) [reply test/e2e/edu-1990-rjsf-helpers.js]: They run successfully on Sauce for IE.

**Pull request merged**